### DISCUSSION TITLE
关于计算器的输出

### DISCUSSION THREAD

**Discussion by ichab** [liusy58/NJU_AdvancedProgramming_SP22#133] [Q&A] [answered]
请问下无论最终结果是整数还是小数都是统一保留一位小数吗

**Comment by binbin00** [comment, marked as answer]
是的

**Reply by VansamaZWT** [reply]
![image](https://user-images.githubusercontent.com/100075314/158534826-2cab17e0-faa1-4ba2-b9c1-395aacf1975d.png)
这是否与FAQ中给到的不太一致

**Reply by binbin00** [reply]
抱歉，FAQ是另一位热心助教写的。既然这样，那两种都可以。
